test/infamy/env.py

```diff
@@ -36,14 +37,21 @@ def __init__(self, ltop=None, argv=sys.argv[1::], environ=os.environ):
         pdot = pydot.graph_from_dot_file(self.args.ptop[0])[0]
         self.ptop = topology.Topology(pdot)
 
+        if self.args.ltop is None:
+            stack = inspect.stack()
+            caller_frame = stack[1]
+            top_path = caller_frame.filename
+            top_path = os.path.join(os.path.dirname(top_path), "topology.dot")
+        else:
+            top_path = self.args.ltop
+
         self.ltop = None
-        if self.args.ltop:
-            ldot = pydot.graph_from_dot_file(self.args.ltop)[0]
-            self.ltop = topology.Topology(ldot)
-            if not self.ltop.map_to(self.ptop):
-                raise tap.TestSkip()
+        ldot = pydot.graph_from_dot_file(top_path)[0]
+        self.ltop = topology.Topology(ldot)
+        if not self.ltop.map_to(self.ptop):
+            raise tap.TestSkip()
 
-            print(repr(self.ltop))
+        print(repr(self.ltop))
```

Comment:
This was previous after a raise, and therefore never executed. It can safely be removed